### PR TITLE
docs: add code example for input-required

### PR DESCRIPTION
**Pull request by gloriliale**
Added a code preview example of a required input component at the Required Field Indicator heading of the Input docs.

### CODE REVIEW

**Review by nkrantz — APPROVED** [review]
Woohoo first PR! Nice use of LivePreview, the placement makes sense too.

File: packages/paste-website/src/pages/components/input/index.mdx [hunk 1]
```diff
@@ -309,6 +309,14 @@ Give users enough information in help text to prevent input and formatting error
 
 Ask for information only when needed. Consider removing the field otherwise.
 
+<LivePreview scope={{Input, Label, HelpText}} language="jsx">
+  {`<>
+  <Label htmlFor="email_address" required>Email Address</Label>
+  <Input aria-describedby="email_help_text" id="email_address" name="email_address" type="email" placeholder="example@twilio.com" onChange="" required/>
+  <HelpText id="email_help_text">Please enter a valid email.</HelpText>
+</>`}
+</LivePreview>
```
Comment (richbachman) [inline packages/paste-website/src/pages/components/input/index.mdx:318]: Nice! 🔥

File: packages/paste-website/src/pages/components/input/index.mdx [hunk 2]
```diff
@@ -309,6 +309,14 @@ Give users enough information in help text to prevent input and formatting error
...
+
 Use required indicators to show users which fields they must fill out.
```
Comment (richbachman) [inline packages/paste-website/src/pages/components/input/index.mdx:320]: Would this sentence fit better above the example? So we explain it, then show it.

**Review by TheSisb — APPROVED** [review]

**Review by richbachman — APPROVED** [review]
@gloriliale will you also create a ticket to update the TextArea doc page with a similar example? Thanks!

**Pull request merged**